mod/wikirate_source/set/type/source/source_preview.rb

```diff
@@ -94,7 +130,7 @@
               <span class="icon-bar"></span>
             </button>
             <div id="source-preview-tabs" class="navbar-brand" href="#">
-              #{web_link("/", :text=>raw( nest Card["*logo"], :view=>:core, :size=>:original ))}
+              #{web_link("/", text: raw( nest Card["*logo"], view: :core, size: :original ))}
```

Comment:
Line is too long. [93/80]<br>Add parentheses to nested method call `nest Card["*logo"], view: :core, size: :original`.<br>Space inside parentheses detected.<br>Prefer single-quoted strings inside interpolations.